common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4036,6 +4039,15 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64k")
 
+  val PARTITION_READER_CHECKPOINT_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.partition.reader.checkpoint.enabled")
+      .categories("client")
+      .version("0.6.0")
```

Comment:
nit: should be 0.5.5, also need change main branch to 0.5.5

Reply:
Ack

Reply:
I've raised it against `[branch-0.5](https://github.com/apache/celeborn/tree/branch-0.5)` Is that okay or is there some other branch specifically for 0.5.5?

Reply:
> I've raised it against `[branch-0.5](https://github.com/apache/celeborn/tree/branch-0.5)` Is that okay or is there some other branch specifically for 0.5.5?

Thanks, It's ok